internal/contact/email.go

```diff
@@ -0,0 +1,43 @@
+package contact
+
+import (
+	"context"
+	"fmt"
+	"github.com/CollActionteam/collaction_backend/internal/constants"
+	"github.com/CollActionteam/collaction_backend/internal/models"
+)
+
+const separator = "### app version:"
+
+type EmailRepository interface {
```

Comment:
What do you think in create a repository layer and put all files there?

Reply:
again based on the above assumption, this logic dictates the requirement, this service need this method only,
so in testing it we only need to mock that, not all the methods.